finminer/FinMiner-linux-2.4.6/run.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -ax
+
+export GPU_FORCE_64BIT_PTR=1
+export GPU_USE_SYNC_OBJECTS=1
+export GPU_MAX_ALLOC_PERCENT=100
+export GPU_MAX_HEAP_SIZE=100
+export GPU_SINGLE_ALLOC_PERCENT=100
+
+# monkey-patching for NVIDIA GPU ᕕ( ಠ_ಠ )ᕗ
+if [[ -d "/usr/local/nvidia/" ]]; then
+    # TODO: remove after debug
+    echo "NVIDIA libs is mounted via SONM GPU plugin, applying patch..."
+    mkdir -p /lib64/nvidia/
+    ln -s /usr/local/nvidia/lib64/libnvidia-ml.so.1 /lib64/nvidia/libnvidia-ml.so
+    export LD_LIBRARY_PATH=/usr/local/nvidia/lib64/:/lib64/nvidia/
+fi
+
+# setting finminer config with envs
+sed -i "s/wallet=WALLET/wallet=$WALLET/" config.ini
```

Comment:
Marked as required in the `task.yaml` example but not really required here. Consider empty variable checking ([example](https://github.com/sonm-io/allowed-list/blob/master/monero-amd/run.sh#L15)).

Reply:
miner fails to start and throw error in logs if wallet is empty, check in run.sh is redundant